### ISSUE TITLE
Doxygen @retval not recognised

### ISSUE THREAD

**Issue by parsley72** [microsoft/vscode-cpptools#6816]
**Type: LanguageService**

**Describe the bug**
- OS and Version: Ubuntu 20.04.01
- VS Code Version: 1.52.1
- C/C++ Extension Version: 1.1.3
- Other extensions you installed (and if the issue persists after disabling them):
- Does this issue involve using SSH remote to run the extension on a remote machine?: No
- A clear and concise description of what the bug is, including information about the workspace (i.e. is the workspace a single project or multiple projects, size of the project, etc).

**Steps to reproduce**

The return values aren't displayed:
```
/**
 * Example function.
 *
 * @param src: [in] Source string.
 * @param key: [in] Key to search for.
 *
 * @retval 0 on success.
 * @retval -1 if \p key not found.
 * @retval -2 if string not terminated properly.
 * @retval -3 if value can't be read.
 */
static int parse_info(const char* ptr, const char* key);
```
![image](https://user-images.githubusercontent.com/1666601/105110830-623c4780-5b24-11eb-84d0-6243c52819d8.png)

**Expected behavior**

A basic function such as this is displayed correctly:
```
/**
 * Example function.
 *
 * @param src: [in] Source string.
 * @param key: [in] Key to search for.
 *
 * @return 0 on success, -1 if \p key not found, -2 if string not terminated properly, -3 if value can't be read.
 */
static int parse_info(const char* ptr, const char* key);
```
![image](https://user-images.githubusercontent.com/1666601/105110813-59e40c80-5b24-11eb-8a95-bd9b92a78d6a.png)

<details>
  <summary><strong>Logs</strong></summary>

```
Insert logs here.
```
</details>

**Screenshots**

**Additional context**

**Comment by TSonono** [comment]
I am able to reproduce this on MacOS 11.1 and VSCode 1.52.1

**Comment by michelleangela** [comment]
By default, only certain Doxygen tags are shown to simplify the quick info view. You can enable showing all tags by setting "C_Cpp.simplifyStructuredComments to "false"

![image](https://user-images.githubusercontent.com/38734287/105226006-43ce5000-5b14-11eb-8378-1ed0ee411d3f.png)

When all Doxygen tags are shown, it should look like:
![image](https://user-images.githubusercontent.com/38734287/105225985-3e710580-5b14-11eb-95f9-7e63cbf3da93.png)

**Comment by TSonono** [comment]
It seems weird that it would not show any of the return values in the "simplified" quick info view. Furthermore, if you disable the simplified quick info view, you don't get the nice structure of "Parameters:" and "Returns:"

**Comment by michelleangela** [comment]
@TSonono We'll add `@retval` to the simplified view similar to `@returns`.

**Comment by sean-mcmanus** [comment]
Fixed with https://github.com/microsoft/vscode-cpptools/releases/tag/1.3.0-insiders

**Comment by michelleangela** [comment]
Fix is released in the 1.3.0 release version https://github.com/microsoft/vscode-cpptools/releases/tag/1.3.0
